examples/data_types_spectra/README.txt

```diff
@@ -0,0 +1,6 @@
+.. _data_types_spectra_examples:
```

Comment:
We are moving spectra into the radiospectra package from this release, and we don't have any examples anyway so I think we can remove this.